modules/spring-config/src/main/resources/META-INF/mule-module.xsd

```diff
@@ -87,6 +67,58 @@
         </xsd:complexContent>
     </xsd:complexType>
 
+    <xsd:element name="parameter" type="parameterType">
+        <xsd:annotation>
+            <xsd:documentation>
+                An named element that will be consumed only in the scope of the body element by the list of message processors.
```

Comment:
why not?
a `<parameter>` is the only visible thing any message processor within the `<body>` element can consume as they don't see the event going through

Mind recommending me a different doc for it?